SPECS-EXTENDED/rust-coreutils/rust-coreutils.signatures.json

```diff
@@ -0,0 +1 @@
+{"Signatures": {"cargo_config.toml": "77e9219c27274120197571fd165cbe4121963b5ad3bc0b20b383c86ef0ce6c2b", "rust-coreutils-0.0.26-vendored.tar.gz": "589d1979c0cedc7a5a9b7f3b260fea0e474dc06ad572fe069b74073642ea43a6", "rust-coreutils-0.0.26.tar.gz": "2a0e8511f1e6adf7f1003ce4536b8a8bb1b2289364efe55edf96f2fc9e2f00a4"}}
```

Comment:
nit - align the json
see this as a reference - https://github.com/microsoft/azurelinux/blob/3.0/SPECS/kernel/kernel.signatures.json
also, the toolkit can help you with this (see `SRPM_FILE_SIGNATURE_HANDLING=update` option) https://github.com/microsoft/azurelinux/blob/4ca9366fb009593d52012d96ac3ac910f1e2377f/CONTRIBUTING.md?plain=1#L190C7-L192